packages/app/features/Send/InformationInput/index.tsx

```diff
@@ -31,9 +33,11 @@ const InformationInput: FC<Props> = ({ navigator }) => {
 		}
 	};
 
+	const { handleClose } = useSnapshot(injectedElements);
+
 	return (
 		<View style={styles.container}>
-			<Header />
+			<ModalHeader content={'Send'} onPressClose={handleClose} />
```

Comment:
string-based props `content={'Send'}` should be written this way: `content="Send"`